Parsing_onliner_project/parsing.py

```diff
@@ -0,0 +1,76 @@
+"""Storage module for class OnlinerHTMLParser"""
```

Comment:
Folders and files could not contain with Capital letters.
It should be named as "parsers" and file itself should be "onliner_parser.py"